src/main/java/de/tum/in/www1/artemis/service/scheduled/TelemetryService.java

```diff
@@ -0,0 +1,90 @@
+package de.tum.in.www1.artemis.service.scheduled;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_SCHEDULING;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.event.EventListener;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+@Service
+@Profile(PROFILE_SCHEDULING)
```

Comment:
Why do you need the scheduling profile here?

Reply:
I'd assume it's so the telemetry service only runs on the "main" node in multi-node setups

Reply:
I removed the profile